docs/en/latest/plugins/dubbo-proxy.md

```diff
@@ -23,40 +30,39 @@ title: dubbo-proxy
 
 ## Description
 
-dubbo-proxy plugin allows you proxy HTTP request to [**dubbo**](http://dubbo.apache.org).
+The `dubbo-proxy` Plugin allows you to proxy HTTP requests to [Apache Dubbo](https://dubbo.apache.org/en/index.html).
+
+:::info IMPORTANT
 
-## Requirement
+If you are using OpenResty, you need to build it with Dubbo support. See [How do I build the APISIX base environment](./../FAQ.md#how-do-i-build-the-apisix-base-environment) for details.
```

Comment:
```suggestion
If you are using OpenResty, you need to build it with Dubbo support. See [How do I build the APISIX base environment](./../FAQ.md#how-do-i-build-the-apisix-base-environment?) for details.
```

Reply:
@hf400159 The "?" does not come in the URL it seems... See: https://apisix.apache.org/docs/apisix/FAQ#how-do-i-build-the-apisix-base-environment